daemon/oci_linux.go

```diff
@@ -463,11 +463,9 @@ func setMounts(daemon *Daemon, s *specs.Spec, c *container.Container, mounts []c
 		}
 
 		if m.Source == "tmpfs" {
-			opt := []string{"noexec", "nosuid", "nodev", volume.DefaultPropagationMode}
+			opt := []string{"noexec", "nosuid", "nodev", volume.DefaultPropagationMode, "size=65536k"}
```

Comment:
Thanks @thaJeztah I just updated the PR to merge the duplicate options. Please let me know if there are any issues.